templates/adspot/handlers/manage.ts

```
@@ -0,0 +1,150 @@
+                                { type: 'p', props: { children: `Amount: ${acceptedBid.amount}` } }
+                            ]
+                        }
+                    },
+                    handler: 'manage',
+                    storage,
+                    webhooks: [{ event: 'adChanged', data: storage.currentAd }],
+                };
+            } else {
+                throw new Error('Invalid bidder');
+            }
+        }
```

Biome

[error] This else clause can be omitted because previous branches break early.

Unsafe fix: Omit the else clause.


(lint/style/noUselessElse)